src/koheesio/models/sql.py

```diff
@@ -28,8 +29,8 @@ class SqlBaseStep(Step, ExtraParamsMixin, ABC):
     sql: Optional[str] = Field(default=None, description="SQL script to apply")
     params: Dict[str, Any] = Field(
         default_factory=dict,
-        description="Placeholders (parameters) for templating. These are identified with ${placeholder} in the SQL "
-        "script. Note: any arbitrary kwargs passed to the class will be added to params.",
+        description="Placeholders (parameters) for templating. The substitutions are identified by braces ('{' and '}')"
+        "and can optionally contain a $-sign. Note: any arbitrary kwargs passed to the class will be added to params.",
```

Comment:
I'm not sure if the current implementation matches the description (as we uncommented the replace params part)

Reply:
We still have: `query = query.replace(f"${{{key}}}", value)` .  So looks like we are good from comment side